release_notes/v2.5.1-beta1.txt

```diff
@@ -0,0 +1,13 @@
+v2.5.1-beta1
```

Comment:
Usual practice is to have a specific PR To update the release-notes etc...

Reply:
The fabric repo release pipeline requires a release_notes/vX.Y.Z manifest to be available at the time of a build trigger.  (This can be annoying when testing updates to the release pipeline...)

I'll remove the -beta1 release notes - this can come along in a later PR when we are ready to publish a build.